draft-ietf-tls-tls13.md

```diff
@@ -753,14 +753,14 @@ Auth | {CertificateVerify*}
 {: #tls-full title="Message flow for full TLS Handshake"}
 
 The handshake can be thought of as having three phases, indicated
-in the diagram above.
+in the diagram above:
```

Comment:
I might put "indicated in the diagram above" in parentheses, as otherwise the colon right after "above" could be a touch jarring.